tool/tctl/common/auth_command.go

```diff
@@ -713,10 +713,11 @@ func (a *AuthCommand) generateUserKeys(ctx context.Context, clusterAPI auth.Clie
 		return nil
 	}
 	if reqExpiry.Sub(expires) > time.Minute {
-		log.Warnf("Requested TTL of %s was not granted. User may have a role with a shorter max session TTL"+
-			" or an existing session ending before the requested TTL. Proceeding with %s",
+		max_allowable_ttl := time.Until(expires).Round(time.Second)
```

Comment:
Almost there. Let's just remove the underscores to make this a more idiomatic variable name.
```suggestion
		maxAllowedTTL := time.Until(expires).Round(time.Second)
```

Reply:
fixed on https://github.com/gravitational/teleport/pull/15384/commits/9e280f31718c90239f8d5e391301002a584c8dff to make variable name more idiomatic